### PR TITLE
[17.0][MIG] base_m2m_custom_field: Migration to 17.0

### PR DESCRIPTION
**Pull request by trisdoan**

### CODE REVIEW

**Review by leemannd — APPROVED** [review]

**Review by chrisb-c01 — APPROVED** [review]

**Comment by chrisb-c01** [comment]
Hi @pedrobaeza, any chance to get this merged? Btw is there a maintainers group for server-tools that we can tag?

File: base_m2m_custom_field/readme/CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- Akim Juillerat \<<akim.juillerat@camptocamp.com>\>
+- Fernando La Chica \<<fernandolachica@gmail.com>\>
```
Comment (pedrobaeza) [inline base_m2m_custom_field/readme/CONTRIBUTORS.md]: Why adding this contributor?

**Review by pedrobaeza — COMMENTED** [review]
You can check here the PSC:

https://oca.github.io/repo-maintainer-conf/server.html#tools-for-odoo-administrators-to-improve-some-technical-features-on-odoo

File: base_m2m_custom_field/readme/CONTRIBUTORS.md (hunk 1 above)
Comment (chrisb-c01) [reply base_m2m_custom_field/readme/CONTRIBUTORS.md]: @trisdoan can you comment or fix this? Then we can get this merged.

**Pull request merged**